src/ProgressCircle.xml

```diff
@@ -4,16 +4,74 @@
+            <enumerationValues>
+                <enumerationValue key="attribute">Attribute</enumerationValue>
+                <enumerationValue key="static">Static</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="progressAttribute" type="attribute" required="false">
+            <caption>Value attribute</caption>
+            <category>Values</category>
+            <description>When no attribute is provided the 'Value static' will be used to calculate the progress</description>
+            <attributeTypes>
+                <attributeType name="Integer"/>
+                <attributeType name="Decimal"/>
+                <attributeType name="Long"/>
+            </attributeTypes>
+        </property>
+        <property key="staticValue" type="integer" defaultValue="50">
+            <caption>Value static</caption>
+            <category>Values</category>
+            <description/>
+        </property>
+        <property key="maximumValueAttribute" type="attribute" required="false">
+            <caption>Maximum value attribute</caption>
+            <category>Values</category>
+            <description>Holds the highest progress value which must be greater than 0. If not set, the maximum value defaults to 'Maximum value static'</description>
+            <attributeTypes>
+                <attributeType name="Integer" />
+                <attributeType name="Decimal" />
+                <attributeType name="Long" />
+            </attributeTypes>
+        </property>
+        <property key="staticMaximumValue" type="integer" defaultValue="100">
+            <caption>Maximum value static</caption>
+            <category>Values</category>
+            <description/>
+        </property>
         <property key="displayText" type="enumeration" defaultValue="percentage">
             <caption>Display text</caption>
-            <category>Appearance</category>
+            <category>Display text</category>
             <description>Text in the progress circle</description>
             <enumerationValues>
                 <enumerationValue key="none">None</enumerationValue>
                 <enumerationValue key="value">Value</enumerationValue>
                 <enumerationValue key="percentage">Percentage</enumerationValue>
+                <enumerationValue key="static">Static</enumerationValue>
+                <enumerationValue key="attribute">Attribute</enumerationValue>
             </enumerationValues>
         </property>
+        <property key="displayTextStatic" type="translatableString" required="false">
+            <caption>Text static</caption>
+            <category>Display text</category>
+            <description>For custom text when display text option is 'static', use {value} to represent the 'staticValue' and {maximumValue} for the 'staticMaximumValue'</description>
+            <translations>
+                <translation lang="en_US">{value} of {maximumValue}</translation>
```

Comment:
Just leave it blank... add this in the description as an example

Reply:
Done

Reply:
Double check with Danny if this feature should be in or should only be possible via attiribute

Reply:
If templating should be supported... than `{percentageRounded0}` `{percentageRounded1}` `{percentageRounded2}`   should also be added rounded on 0 digits, 1 digit and 2 digit? Please double check with Danny
Maybe `{percentage}` rounded with 0 digit should be sufficient....